proto/stride/auction/properties.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+package stride.auction;
+
+import "google/protobuf/any.proto";
+import "gogoproto/gogo.proto";
+import "stride/auction/params.proto";
+
+
+
+option go_package = "github.com/Stride-Labs/stride/v5/x/auction/types";
+
+/* ****************************************************************************
+Properties are used to set unchanging aspects of auction pools and auctions
+
+In contrast to params which are module-wide defaults, properties are specific
+to the new pool or auction being created.
+
+Properties remained fixed for the life of the pool or auction whereas the 
+other fields represent changing state
+**************************************************************************** */
+
+enum AuctionType {
+    UNKNOWN = 0;
+    ASCENDING = 1;
+    DESCENDING = 2;
+    SEALEDBID = 3;
+}
+
+message AscendingAuctionProperties {
+    uint64 supply = 1;
+    uint64 minAllowedBid = 2;
+    uint64 maxAllowedBid = 3;
+    float redemptionRate = 4;
```

Comment:
Please don't use float for proto messages since float is not safety for managing decimals.
Instead, please use sdk decimal type.
example,
string redemption_rate = 4 [
    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
    (gogoproto.nullable) = false
]